deployment/src/main/java/io/quarkiverse/web/bundler/deployment/PrepareForBundlingProcessor.java

```diff
@@ -92,7 +112,7 @@ ReadyForBundlingBuildItem prepareForBundling(WebBundlerConfig config,
             LiveReloadBuildItem liveReload,
             LaunchModeBuildItem launchMode,
             BuildProducer<HotDeploymentWatchedFileBuildItem> watchedFiles,
-            HttpBuildTimeConfig httpBuildTimeConfig,
+            VertxHttpBuildTimeConfig httpBuildTimeConfig,
```

Comment:
I think that we should consume the `io.quarkus.vertx.http.deployment.HttpRootPathBuildItem` instead...

Reply:
Haa ok it is not documented in the `VertxHttpBuildTimeConfig ` :-/

Reply:
Thanks!

Reply:
Ok, to get the compress build media types and all you need the full config

Reply:
Yeah, I think that many extensions expect the config to be part of the public API but there is no consensus on this AFAIK.